py-polars/tests/unit/io/test_database_write.py

```diff
@@ -23,12 +23,16 @@ def adbc_sqlite_driver_version(*args: Any, **kwargs: Any) -> str:
     return "n/a"
 
 
-@pytest.mark.write_disk()
-@pytest.mark.parametrize("engine", ["adbc", "sqlalchemy"])
+@pytest.mark.skipif(
+    sys.version_info > (3, 11),
+    reason="connectorx cannot be installed on Python 3.12 yet.",
+)
```

Comment:
It seems weird that this test relies on both connectorx _and_ adbc.. is there a bug here? 🤔

Reply:
Don't think so; `write_database` support is being tested against each engine.